icomoon/style.css

```diff
@@ -0,0 +1,60 @@
+@font-face {
+  font-family: 'icomoon';
+  src:  url('fonts/icomoon.eot?5jrnyf');
+  src:  url('fonts/icomoon.eot?5jrnyf#iefix') format('embedded-opentype'),
+    url('fonts/icomoon.ttf?5jrnyf') format('truetype'),
+    url('fonts/icomoon.woff?5jrnyf') format('woff'),
+    url('fonts/icomoon.svg?5jrnyf#icomoon') format('svg');
```

Comment:
You need only woff. Do not forget update paths.

Reply:
The same question. Which path should I specify if the fily icomoon/style.css is deleted and the styles are moved to ./styles/_icomoon.scss

Reply:
Updated the path. Everything works now